vhdbuilder/packer/components.json

```diff
@@ -220,6 +220,7 @@
         "v1.24.3",
         "v1.24.4",
         "v1.26.3-2",
+        "v1.26.3-3",
```

Comment:
`azurefile-csi:v1.26.3-3` has CVE, and `azurefile-csi:v1.26.4` is good without CVE, let's close this cve patch, thanks.
```
# trivy image --ignore-unfixed mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.26.3-3
2023-08-04T09:33:31.188Z        INFO    Vulnerability scanning is enabled
2023-08-04T09:33:31.188Z        INFO    Secret scanning is enabled
2023-08-04T09:33:31.188Z        INFO    If your scanning is slow, please try '--security-checks vuln' to disable secret scanning
2023-08-04T09:33:31.188Z        INFO    Please see also https://aquasecurity.github.io/trivy/v0.36/docs/secret/scanning/#recommendation for faster secret detection
2023-08-04T09:33:31.914Z        INFO    Detected OS: alpine
2023-08-04T09:33:31.914Z        INFO    Detecting Alpine vulnerabilities...
2023-08-04T09:33:31.917Z        INFO    Number of language-specific files: 1
2023-08-04T09:33:31.917Z        INFO    Detecting gobinary vulnerabilities...

mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.26.3-3 (alpine 3.15.1)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)


azurefileplugin (gobinary)

Total: 3 (UNKNOWN: 0, LOW: 0, MEDIUM: 3, HIGH: 0, CRITICAL: 0)

âââââââââââââââââââââ¬ââââââââââââââââ¬âââââââââââ¬ââââââââââââââââââââ¬âââââââââââââââââââââââââââââââââââ¬ââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââ
â      Library      â Vulnerability â Severity â Installed Version â          Fixed Version           â                           Title                           â
âââââââââââââââââââââ¼ââââââââââââââââ¼âââââââââââ¼ââââââââââââââââââââ¼âââââââââââââââââââââââââââââââââââ¼ââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââ¤
â k8s.io/kubernetes â CVE-2023-2431 â MEDIUM   â v1.26.0           â 1.27.2, 1.26.5, 1.25.10, 1.24.14 â Bypass of seccomp profile enforcement                     â
â                   â               â          â                   â                                  â https://avd.aquasec.com/nvd/cve-2023-2431                 â
â                   âââââââââââââââââ¤          â                   ââââââââââââââââââââââââââââââââââââ¼ââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââ¤
â                   â CVE-2023-2727 â          â                   â 1.24.15, 1.25.11, 1.26.6, 1.27.3 â Bypassing policies imposed by the ImagePolicyWebhook      â
â                   â               â          â                   â                                  â admission plugin                                          â
â                   â               â          â                   â                                  â https://avd.aquasec.com/nvd/cve-2023-2727                 â
â                   âââââââââââââââââ¤          â                   â                                  âââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââ¤
â                   â CVE-2023-2728 â          â                   â                                  â Bypassing enforce mountable secrets policy imposed by the â
â                   â               â          â                   â                                  â ServiceAccount admission plugin...                        â
â                   â               â          â                   â                                  â https://avd.aquasec.com/nvd/cve-2023-2728                 â
âââââââââââââââââââââ´ââââââââââââââââ´âââââââââââ´ââââââââââââââââââââ´âââââââââââââââââââââââââââââââââââ´ââââââââââââââââââââââââââââââââââââââââââââââââââââââââââââ
```

Reply:
```
# trivy image --ignore-unfixed mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.26.4
2023-08-04T09:34:31.982Z        INFO    Vulnerability scanning is enabled
2023-08-04T09:34:31.982Z        INFO    Secret scanning is enabled
2023-08-04T09:34:31.982Z        INFO    If your scanning is slow, please try '--security-checks vuln' to disable secret scanning
2023-08-04T09:34:31.982Z        INFO    Please see also https://aquasecurity.github.io/trivy/v0.36/docs/secret/scanning/#recommendation for faster secret detection
2023-08-04T09:34:32.177Z        INFO    Detected OS: alpine
2023-08-04T09:34:32.177Z        INFO    Detecting Alpine vulnerabilities...
2023-08-04T09:34:32.185Z        INFO    Number of language-specific files: 1
2023-08-04T09:34:32.185Z        INFO    Detecting gobinary vulnerabilities...

mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.26.4 (alpine 3.15.1)

Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)

```

Reply:
glad to know it! thanks for sharing this!